src/view/dataElements/xdmObject/helpers/schemaSelection/loadDefaultSandbox.js

```diff
@@ -27,15 +28,21 @@ const loadDefaultSandbox = async ({
       imsAccess
     }));
   } catch (e) {
-    console.error(e);
-    reportAsyncError(new Error(`Failed to load sandboxes. ${e.message}`));
+    reportAsyncError(
+      new UserReportableError("Failed to load sandboxes.", {
+        originatingError: e
+      })
+    );
     return undefined;
   }
 
   if (!sandboxes.length) {
     reportAsyncError(
-      new Error(
-        "You do not have access to any sandboxes. Please contact your administrator to be assigned appropriate rights."
+      new UserReportableError(
+        "You do not have access to any sandboxes. Please contact your administrator to be assigned appropriate rights.",
```

Comment:
I think I can add some actionability on a couple of the others. Most errors though, particularly the non-user-reportable errors, don't seem to have anything user-actionable. For example, if we have some sort of unforeseen JavaScript error or the server blew up and is spewing 500 status codes, I don't think there's anything a customer can do to fix the problem. I don't have confidence that refreshing the Launch page or clicking buttons will help, so "try again later" is just that--try again sometime later because hopefully by that point we've fixed the problem on our side. The one actionable thing they could probably do is report the issue through their customer rep. Do you have any opinion on what it should say in these scenarios?

Reply:
BTW, I logged this issue so that we're more proactive at detecting and fixing errors occurring for customers: https://jira.corp.adobe.com/browse/PDCL-6348

Reply:
I like the ticket you created. That is much better than giving the user a link to report the error. I'm glad you looked through the errors.